torch/testing/_internal/common_utils.py

```diff
@@ -1747,7 +1747,7 @@ def remove_device_and_dtype_suffixes(test_name: str) -> str:
 def check_if_enable(test: unittest.TestCase):
     test_suite = str(test.__class__).split('\'')[1]
     raw_test_name = f'{test._testMethodName} ({test_suite})'
-    if raw_test_name in slow_tests_dict:
+    if raw_test_name in slow_tests_dict.get(os.getenv("BUILD_ENVIRONMENT", ""), {}):
```

Comment:
Do you want to keep the values from `slow-tests.json` as fallback values when the `BUILD_ENVIRONMENT` or job name couldn't be found in the new `slow-tests-by-job.json` file?